craterpy/exceptions.py

```diff
@@ -0,0 +1,17 @@
+"""Custom exceptions used by this project."""
+
+
+class CraterAttributeError(AttributeError):
```

Comment:
Please suggest a better name for this (and CraterValueError). Otherwise this is just a pretty label around the parent exception.

Reply:
Re: your comment on #5, we can probably scrap the ones that simply wrap AttributeError and ValueError